src/main/java/Objects/GameMapImpl.java

```diff
@@ -349,14 +351,17 @@ private synchronized void nextRound() {
             if(captureAwardMineral) {
                 final long amountMined = mine.mineResources(Configurations.getMineResourceAmount());
                 userMinerals.put(mine.getOwner().getName(), Math.min(userCapacity.get(mine.getOwner().getName()), amountMined + userMinerals.get(mine.getOwner().getName())));
+
+                System.err.println(mine.getResources());
             }
         });
         mines.stream().forEach((mine) -> {
             mine.replenishResources(Configurations.getMineResourceReplenishAmount());
         });
 
+
         Set<Bomb> removeBombs = new CopyOnWriteArraySet<>();
-        bombs.stream().parallel().forEach(bomb -> {

```

Comment:
What's the reason for removing this?
Shouldn't matter, but I'm curious about it because I think I left it in the other code I wrote.